js/wp-seo-post-scraper.js

```diff
@@ -2,6 +2,10 @@
 (function( $ ) {
 	'use strict';
 
+	if ( typeof ckeditorOn === 'function' ){
+		console.warn( 'YoastSEO currently doesn\'t support ckEditor' );
```

Comment:
YoastSEO currently doesn\'t support ckEditor. The content analysis might not function optimally.